.github/workflows/build-and-test-windows.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: Install-WindowsFeature -name Web-Server -IncludeManagementTools
       - uses: actions/setup-go@v4
         with:
-          go-version: ~1.19.10
+          go-version: ~1.19.11
```

Comment:
Could this be handled by dropping the patch number?  If we only specify `1.19` that will grab the latest patch version on each run right, preventing us from needing to do these bump PRs for CVE patches in the future?

Reply:
The [tilde](https://github.com/npm/node-semver#tilde-ranges-123-12-1) already allows that. Bumping the patch version is a formality and provides an ease of mind to ensure that we can guarantee a minimum version.

Reply:
I think™ the reason that the latest version is not selected has to do with the [`setup-go`](https://github.com/actions/setup-go) logic in selecting the go version to use. It looks at the cache first to see if a version exists in the range and only downloads when necessary. I cannot confirm this but it would explain the behavior.

Reply:
We could set [`check-latest`](https://github.com/actions/setup-go#check-latest-version) to true though. I think that would also be another guarantee.

Reply:
The concern I have with only using `1.19` is that it makes no guarantee that an earlier patch release won't be used.  While probably not critical here since we do not produce release artifacts from these workflows, it can lead to flaky tests due to the `govulncheck` linter.